docs/tutorials/wiki2/src/models/tutorial/__init__.py

```diff
@@ -5,8 +5,6 @@ def main(global_config, **settings):
     """ This function returns a Pyramid WSGI application.
     """
     config = Configurator(settings=settings)
-    settings = config.get_settings()
```

Comment:
Heh, yeah, this made me do a double-take, too. Notice the directory structure. These lines appear one directory deeper in the hierarchy within `project_name/models/__init__.py`, at `docs/tutorials/wiki2/src/*/tutorial/models/__init__.py` for each of the seven steps of the tutorial and the alchemy cookiecutter at https://github.com/Pylons/pyramid-cookiecutter-alchemy/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/%7B%7Bcookiecutter.repo_name%7D%7D/models/__init__.py#L59-L60. I assumed it was an accidental addition at some point.